### PR TITLE
Adds example for modifiers.

### PR DESCRIPTION
**Pull request by janzenz**

### CODE REVIEW

File: docs/css.md [hunk 1]
```diff
@@ -54,10 +54,10 @@ However for larger projects with code-splitting, or for dynamic CSS we like [sty
 ```scss
 // Good (Easy to search within the codebase)
 .tile {}
-.tile--blue {}
+.tile--error {}
 .tile__content {}
 .tile__heading {}
-.tile__heading--red {}
+.tile__heading--error {}
```
Comment (katie-day) [inline docs/css.md]: Using `--red/--blue` might be a better example for here. We talk about modifiers for "state" further down here: https://github.com/springload/frontend-starter-kit/blob/f74cc9e0b3f0b0f4ef6764b301967c4fbb076dd0/docs/css.md#misc (just above misc), where we explain if we are modifying the state, we use something like `tile__heading--has-error` which makes it extra clear as to what its doing :)

**Comment by holloway** [comment]
@janzenz @katie-day what should we do this with this PR? Close it?

File: docs/css.md [hunk 2]
```diff
@@ -54,8 +54,10 @@ However for larger projects with code-splitting, or for dynamic CSS we like [sty
 ```scss
 // Good (Easy to search within the codebase)
 .tile {}
+.tile--error {}
```
Comment (janzenz) [inline docs/css.md]:
```suggestion
.tile--has-error {}
```

File: docs/css.md [hunk 3]
```diff
@@ -54,8 +54,10 @@ However for larger projects with code-splitting, or for dynamic CSS we like [sty
 ```scss
 // Good (Easy to search within the codebase)
 .tile {}
+.tile--has-error {}
 .tile__content {}
 .tile__heading {}
+.tile__heading--error {}
```
Comment (janzenz) [inline docs/css.md]:
```suggestion
.tile__heading--has-error {}
```

**Comment by janzenz** [comment]
@holloway I've done the changes based on @katie-day feedback.

**Pull request merged**